java/client/build.gradle

```diff
@@ -234,3 +235,22 @@ jar {
     // placeholder will be renamed by platform+arch on the release workflow java-cd.yml
     archiveClassifier = "placeholder"
 }
+
+spotbugsMain {
+    reports {
+        html {
+            required = true
+            outputLocation = file("$buildDir/reports/spotbugs/main/spotbugs.html")
+            stylesheet = 'fancy-hist.xsl'
+        }
+        xml {
+            required = true
+            outputLocation = file("$buildDir/reports/spotbugs/main/spotbugs.xml")
+        }
+    }
+}
+
+spotbugs {
+    ignoreFailures = true
```

Comment:
should we include a TODO to fix failures and remove this check?

Reply:
Good question. Let's ensure we track and plan it separately. I suggest we track it as an issue/task rather than do it in the Gradle script.

Reply:
We have a TODO list.  I will track it there.